### PR TITLE
[Feature] Adding Step Function

### PR DESCRIPTION
**Pull request by grahamjenson**

### CODE REVIEW

**Comment by heimdall-asguard** [comment]
Reviews      | 0/1 | Message
------------ | ------------- | ----------




[More info](https://heimdall-public.cbhq.net/v1/code_change?repository=coinbase/geoengineer&ref=111837b6ad14733c8f7f89666611859b7a121821)

**Comment by heimdall-asguard** [comment]
Review Error for jack-kearney@2018-02-22T02:36:25+00:00
User failed mfa authentication

File: lib/geoengineer/resources/aws_sfn_state_machine_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+########################################################################
+# AwsSfnStateMachine is the +aws_sfn_state_machine+ terrform resource,
```
Comment (krobertson) [inline lib/geoengineer/resources/aws_sfn_state_machine_spec.rb]: Mean to put a period rather than a comma?

**Comment by heimdall-asguard** [comment]
Reviews      | 1/1 | Message
------------ | ------------- | ----------
krobertson | 1 | approved 

Pull request is reviewed and can be merged to master!

[More info](https://heimdall-public.cbhq.net/v1/code_change?repository=coinbase/geoengineer&ref=111837b6ad14733c8f7f89666611859b7a121821)

File: lib/geoengineer/resources/aws_sfn_state_machine_spec.rb (hunk 1 above)
Comment (grahamjenson) [reply lib/geoengineer/resources/aws_sfn_state_machine_spec.rb]: No idea.

**Comment by heimdall-asguard** [comment]
Reviews      | 1/1 | Message
------------ | ------------- | ----------
krobertson | 1 | approved 

Pull request is reviewed and can be merged to master!

[More info](https://heimdall-public.cbhq.net/v1/code_change?repository=coinbase/geoengineer&ref=40c181507020f009a2d8b06b4551b0469f5a3343)

**Pull request merged**